rift/common/__init__.py

```diff
@@ -1,5 +1,11 @@
-#very, very magic code to cleanup the immutable schema to allow old code to modify rift packets
-
+#
+#
+# $Id
+#
+# Copyright (c) 2021, Juniper Networks, Inc.
```

Comment:
Can you please remove all "Copyright (c) 2021, Juniper Networks, Inc." and especially all "All rights reserved" lines from the pull request.  Also, can you please send my an e-mail to confirm, in writing, that all of the code in your pull requests is contributed under the Apache 2.0 license (which is the license that rift-python uses).

Reply:
ack